Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -27,6 +27,59 @@
       types:
         Blunt: 0
 
+- type: entity
+  parent: BasePlushie
+  id: PlushieGhost
+  name: ghost soft toy
+  description: The start of your personal GHOST GANG!
+  components:
+    - type: Sprite
+      sprite: Mobs/Ghosts/ghost_human.rsi
+      state: icon
+      netsync: false
+      noRot: true
+    - type: Item
+    - type: Fixtures
+      fixtures:
+        - shape:
+            !type:PhysShapeAabb
+            bounds: "-0.25,-0.25,0.25,0.25"
+          density: 20
+          mask:
+            - ItemMask
+          restitution: 0.98
+          friction: 0.01
+    - type: Physics
+      angularDamping: 0.02
+      linearDamping: 0.02
+      fixedRotation: true
+      bodyType: Dynamic
+    - type: TileFrictionModifier
```

Comment:
These stats make it especially janky because the pickup + following both mispredict a lot.

Reply:
Do you have the fakelag disabled locally? It should be on by default.